hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/create/CreateTranslator.java

```diff
@@ -129,21 +128,17 @@ public CreateTranslator(CreateDecoder decoder) {
 
     @Override
     public boolean matches(@NonNull HtsCallAttempt attempt) {
-        return Arrays.equals(attempt.selector(), CreateTranslator.CREATE_FUNGIBLE_TOKEN_V1.selector())
-                || Arrays.equals(attempt.selector(), CreateTranslator.CREATE_FUNGIBLE_TOKEN_V2.selector())
-                || Arrays.equals(attempt.selector(), CreateTranslator.CREATE_FUNGIBLE_TOKEN_V3.selector())
-                || Arrays.equals(attempt.selector(), CreateTranslator.CREATE_FUNGIBLE_WITH_CUSTOM_FEES_V1.selector())
-                || Arrays.equals(attempt.selector(), CreateTranslator.CREATE_FUNGIBLE_WITH_CUSTOM_FEES_V2.selector())
-                || Arrays.equals(attempt.selector(), CreateTranslator.CREATE_FUNGIBLE_WITH_CUSTOM_FEES_V3.selector())
-                || Arrays.equals(attempt.selector(), CreateTranslator.CREATE_NON_FUNGIBLE_TOKEN_V1.selector())
-                || Arrays.equals(attempt.selector(), CreateTranslator.CREATE_NON_FUNGIBLE_TOKEN_V2.selector())
-                || Arrays.equals(attempt.selector(), CreateTranslator.CREATE_NON_FUNGIBLE_TOKEN_V3.selector())
-                || Arrays.equals(
-                        attempt.selector(), CreateTranslator.CREATE_NON_FUNGIBLE_TOKEN_WITH_CUSTOM_FEES_V1.selector())
-                || Arrays.equals(
-                        attempt.selector(), CreateTranslator.CREATE_NON_FUNGIBLE_TOKEN_WITH_CUSTOM_FEES_V2.selector())
-                || Arrays.equals(
-                        attempt.selector(), CreateTranslator.CREATE_NON_FUNGIBLE_TOKEN_WITH_CUSTOM_FEES_V3.selector());
+        return attempt.isSelector(CREATE_FUNGIBLE_TOKEN_V1, CREATE_FUNGIBLE_TOKEN_V2, CREATE_FUNGIBLE_TOKEN_V3)
```

Comment:
Could be one long list of selectors in one `attempt.isSelector(...)` call, so out of curiousity: Why did you choose this formulation?  I see the methods are "grouped" in a logical way, is that it?

Reply:
It's grouped and easier to read and honestly looks better than (imo) single call with 10+ selectors